packages/server/api/src/app/flows/flow/flow-version.controller.ts

```diff
@@ -197,7 +197,8 @@ const GetLatestVersionsByConnectionRequestOptions = {
     permission: Permission.READ_FLOW,
   },
   schema: {
-    description: 'Retrieves all flows that contain a specific connection',
+    description:
```

Comment:
this endpoint actually confuses me a bit, i dont feel its the right one or am i insane?

Reply:
sadly it is the right one, used when the connection is deleted.